test/integration/bastion/bastion_test.go

```diff
@@ -639,6 +606,5 @@ func getMyPublicIPWithMask() (string, error) {
 	if err != nil {
 		return "", err
 	}
-	//res := strings.Join(strings.Split(string(body), ".")[:3], ".") + ".0/24"
-	return strings.TrimSpace(string(body)) + "/32", nil
+	return strings.Join(strings.Split(string(body), ".")[:3], ".") + ".0/24", nil
```

Comment:
this looks strange. Please use the functions from the `net` package to apply the mask
e.g. https://github.com/gardener/gardenctl-v2/blob/master/pkg/cmd/ssh/options.go#L141-L156

Reply:
@petersutter , 0890d89c  done